### PR TITLE
Changes from linter

### PR DESCRIPTION
**Pull request by dannykopping**
**What this PR does / why we need it**:
We had a few outstanding linter issues, and running `make lint` fixes them all automatically.

### CODE REVIEW

**Review by kavirajk — APPROVED** [review]
LGTM

File: pkg/ruler/memstore.go [hunk 1]
```diff
@@ -3,10 +3,11 @@ package ruler
 import (
 	"context"
 	"errors"
-	"github.com/prometheus/prometheus/model/rulefmt"
 	"sync"
 	"time"
 
+	"github.com/prometheus/prometheus/model/rulefmt"
```
Comment (vlad-diachenko) [inline pkg/ruler/memstore.go:9]: I believe it must be grouped with the group below

File: pkg/ruler/memstore_test.go [hunk 2]
```diff
@@ -2,11 +2,12 @@ package ruler
 
 import (
 	"context"
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/model/rulefmt"
 	"testing"
 	"time"
 
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/rulefmt"
```
Comment (vlad-diachenko) [inline pkg/ruler/memstore_test.go:9]: I believe it must be grouped with the group below

**Review by vlad-diachenko — APPROVED** [review]
LGTM.

File: pkg/ruler/memstore_test.go (hunk 2 above)
Comment (dannykopping) [reply pkg/ruler/memstore_test.go:9]: I'm just sticking with what the linter says

**Pull request merged**